types/node/test.d.ts

```diff
@@ -452,6 +462,30 @@ declare module "node:test" {
      * @since v18.0.0, v16.17.0
      */
     class TestContext {
+        /**
+         * An object containing assertion methods bound to `context`.
+         * The top-level functions from the `node:assert` module are exposed here for the purpose of creating test plans.
+         * @since v22.2.0
+         */
+        assert: {
```

Comment:
This should probably be `readonly`, and might be better spun off to a separate interface, especially as it's due to gain its own methods (`snapshot()` in v22.3). Might also be useful to include brief docblocks. ([example](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/9e65ff4b586b1dd8b4fbe7c9cc7637a7821f91a0#diff-fb23a8f2e7fc5caa3d3e37f70d7138e548925757579ca9e6d100d9dfb83fa20fR610))